x/airdrop/keeper/claim.go

```diff
@@ -159,11 +159,7 @@ func (k Keeper) ClaimEarly(ctx sdk.Context, airdropId, claimer string) error {
 // There's no need to merge the Claimed or Forfeited amounts because the host allocations cannot
 // be claimed through a non-stride address
 func (k Keeper) LinkAddresses(ctx sdk.Context, airdropId, strideAddress, hostAddress string) error {
-	// Fetch the airdrop and user's allocations
```

Comment:
is the thinking behind removing this check that the `hostFound` check below should cover it?

Reply:
it was moved into the main msg_server function!

Reply:
@shellvish can i mark this resolved?

Reply:
Yup, makes sense!